frontend/src/store/index.js

```diff
@@ -0,0 +1,61 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+import axios from 'axios'
+import biri from 'biri'
+import io from 'socket.io-client'
+
+import { mutations, actions } from './types'
+import user from './user'
+import event from './event'
+
+
+Vue.use(Vuex)
+
+axios.defaults.baseURL = process.env.VUE_APP_BASE_PATH
+axios.defaults.withCredentials = true
+
+const socket = io(process.env.VUE_APP_SOCKET_PATH)
+
+const store = new Vuex.Store({
+  state: {
+    loading: false,
+  },
+  mutations: {
+    [mutations.SET_PROPERTY](state, obj) {
+      for (var key in obj) {
+        state[key] = obj[key]
+      }
+    },
+  },
+  actions: {
+    async [actions.SET_PROPERTY]({ commit }, obj) {
+      commit(mutations.SET_PROPERTY, obj)
```

Comment:
this actoin shouldn't exist. it's an anti-pattern, you can fix it while you are at it.